conf/modules.config

```
@@ -263,13 +263,13 @@ process {
     }
 
     withName: METASPADES {
-        ext.args   = params.spades_options ?: '--meta'
+        ext.args   = params.spades_options ? params.spades_options + '--meta' : '--meta'
```

```suggestion
        ext.args   = params.spades_options ? params.spades_options + ' --meta' : '--meta'
```
not sure, but otherwise the two strings would be concatenated without a space inbetween? If thats true, also add below.

---

Whoops good catch!